aspnetcore/grpc/browser.md

```diff
@@ -11,156 +11,22 @@ uid: grpc/browser
-
-To use gRPC-Web:
-
-* Add a reference to the [Grpc.Net.Client.Web](https://www.nuget.org/packages/Grpc.Net.Client.Web) package.
-* Ensure the reference to [Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client) package is 2.29.0 or greater.
-* Configure the channel to use the `GrpcWebHandler`:
-
-[!code-csharp[](~/grpc/browser/sample/Handler.cs?name=snippet_1)]
-
-The preceding code:
-
-* Configures a channel to use gRPC-Web.
-* Creates a client and makes a call using the channel.
-
-`GrpcWebHandler` has the following configuration options:
-
-* **InnerHandler**: The underlying <xref:System.Net.Http.HttpMessageHandler> that makes the gRPC HTTP request, for example, `HttpClientHandler`.
-* **GrpcWebMode**: An enumeration type that specifies whether the gRPC HTTP request `Content-Type` is `application/grpc-web` or `application/grpc-web-text`.
-    * `GrpcWebMode.GrpcWeb` configures content to be sent without encoding. Default value.
-    * `GrpcWebMode.GrpcWebText` configures content to be base64 encoded. Required for server streaming calls in browsers.
-* **HttpVersion**: HTTP protocol `Version` used to set [HttpRequestMessage.Version](xref:System.Net.Http.HttpRequestMessage.Version) on the underlying gRPC HTTP request. gRPC-Web doesn't require a specific version and doesn't override the default unless specified.
-
-> [!IMPORTANT]
-> Generated gRPC clients have sync and async methods for calling unary methods. For example, `SayHello` is sync and `SayHelloAsync` is async. Calling a sync method in a Blazor WebAssembly app will cause the app to become unresponsive. Async methods must always be used in Blazor WebAssembly.
-
-### Use gRPC client factory with gRPC-Web
-
-A gRPC-Web compatible .NET client can be created using the [gRPC client factory](xref:grpc/clientfactory).
-
-To use gRPC-Web with client factory:
-
-* Add package references to the project file for the following packages:
-  * [Grpc.Net.Client.Web](https://www.nuget.org/packages/Grpc.Net.Client.Web)
-  * [Grpc.Net.ClientFactory](https://www.nuget.org/packages/Grpc.Net.ClientFactory)
-* Register a gRPC client with dependency injection (DI) using the generic `AddGrpcClient` extension method. In a Blazor WebAssembly app, services are registered with DI in `Program.cs`.
-* Configure `GrpcWebHandler` using the <xref:Microsoft.Extensions.DependencyInjection.HttpClientBuilderExtensions.ConfigurePrimaryHttpMessageHandler%2A> extension method.
-
-```csharp
-builder.Services
-    .AddGrpcClient<Greet.GreeterClient>(options =>
-    {
-        options.Address = new Uri("https://localhost:5001");
-    })
-    .ConfigurePrimaryHttpMessageHandler(
-        () => new GrpcWebHandler(new HttpClientHandler()));
-```
-
-For more information, see <xref:grpc/clientfactory>.
+> gRPC JSON transcoding requires .NET 7 or later.
```

Comment:
> You explicitly want to make everyone aware of transcoding even if they aren't on 7.0 or considering moving to 7.0 so that they might be encouraged to upgrade or unlock some great features.

^ This is why 😄